src/pages/adopters/anutanetworks.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Layout } from "../../components/layout";
+import { Comments } from "../../components/sections/individual-adopters/Comments";
+import { Hero } from "../../components/sections/individual-adopters/Hero";
+import { Video } from "../../components/sections/individual-adopters/Video";
+import { useLocation } from "@reach/router";
+
+const AnutaNetworks = () => {
+  const location = useLocation();
+  const urlSlug = location.pathname
+    ?.replace(/\/$/, "")
+    .split("/")
+    .pop();
+  return (
+    <Layout noPreFooter>
```

Comment:
you can make these with the createPage API dynamically and use this as template

Reply:
Arko I already did made it using the dynamic createPage Api, but I think this force push to staging mixed up a lot of code.
Im trying to clean everything up